src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -158,7 +166,8 @@ export default class CheckoutPlugin extends Plugin {
     }
 
     onConfirmPayment (event) {
-        if (!this.cardFormValidator.validateForm()) {
+        //TODO Implement validation for multiple PMs
+        if (!this.cardFormValidator.validateForm() && false) {
```

Comment:
I think just the name should be changed since the component does the validation for us with a common validate function

Reply:
Do you think we should do it in this PR? I had created a ticket for that (PW-2914) but we could do it here

Reply:
it's fine in the other ticket :) I will resolve this comment